kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterCondition.java

```diff
@@ -29,14 +33,9 @@ static ClusterCondition accepted(String cluster) {
         return new ClusterCondition(cluster, Condition.Type.Accepted, Condition.Status.TRUE, null, null);
     }
 
-    static ClusterCondition filterInvalid(String cluster, String filterName, String detail) {
-        return new ClusterCondition(cluster, Condition.Type.Accepted, Condition.Status.FALSE, INVALID,
-                String.format("Filter \"%s\" is invalid: %s", filterName, detail));
-    }
-
-    public static ClusterCondition refNotFound(String cluster, LocalRef<?> ref) {
-        return new ClusterCondition(cluster, Condition.Type.ResolvedRefs, Condition.Status.FALSE, INVALID,
-                String.format("Resource of kind \"%s\" in group \"%s\" named \"%s\" does not exist.", ref.getKind(), ref.getGroup(), ref.getName()));
+    public static ClusterCondition refNotFound(HasMetadata referee, LocalRef<?> ref) {
+        return new ClusterCondition(name(referee), Condition.Type.ResolvedRefs, Condition.Status.FALSE, INVALID,
+                String.format("Resource %s does not exist.", coordinatesString(ref, referee)));
```

Comment:
Can we change this to "Resource %s was not found"? That aligns with what's usually the underlying cause -- we got a 404 when we requested the referenced resource.